docs/getting-started/steps-to-build-deploy-contract.md

```diff
@@ -0,0 +1,80 @@
+
+The simulation will help to reveal any errors in your contract such as failing to make a payment. You can also verify that a refund occurs if certain conditions are met. If you do not include a timeout, simulating the contract will help to reveal this. For example, if a participant fails to make a deposit, the contract should include a timeout. 
+
+#### To simulate your contract
+
+1. With your contract open in the Playground, select ‘Send to Simulator’ in the upper right area. A panel opens where you can specify deadline parameters and ada amounts. 
+2. Step through the contract’s execution and test various choices, examining different potential pathways. This will give you insight into how your contract will behave under different conditions. 
+
+## Deploying a smart contract
+
+Use Marlowe Runner to deploy your smart contract to the blockchain. Marlowe Runner streamlines and simiplifies the process of deploying smart contracts on Cardano. It eliminates the need for low-level tools and deploying your own backend Runtime instance. 
+
+#### To deploy your contract
+
+1. With your contract open in the Playground, select ‘Send to Simulator,’ then ‘Export to Marlowe Runner.’ A window displays prompting you to select either the pre-production or preview network for deployment. After you make your selection, Playground generates and exports the JSON file directly to Marlowe Runner. Runner automatically displays in a browser window and prompts you to choose a wallet. 
+2. Make your wallet selection. Make sure that your wallet is connected to the same network that you selected for your contract deployment. Runner displays the source code for your contract. 
+3. In the ‘Add tags’ field, enter a name for your contract. 
+4. Click ‘Submit contract.’ Depending on the details of your contract, you may be prompted to specify addresses for borrower and lender, or you may be prompted to confirm the transaction with your wallet. 
+5. Follow the prompts. The Runner displays a list view that shows the current status of your contract. The status shown under the Actions column will update as your contract progresses. 
+6. When ‘Advance’ displays under the Action column for your contract, select ‘Advance’ to proceed. The Source graph page displays. The source graph will show the possible execution paths of your contract. 
+7. Below the source graph, the next available action will be reflected, such as a deposit, for example. Runner is designed to prompt you through the execution of your contract. 
+8. Continue to follow the prompts, using your wallet password to authorize transactions for your role. 
+
+From the Runner page that lists pending transactions, when the 'Action' column indicates a status of ‘Complete,’ select the Contract Id link to view your transaction in the source graph page. It will show the execution path of your contract. 
```

Comment:
```suggestion
From the Runner page that lists pending transactions, when the 'Action' column indicates a status of ‘Complete,’ select the contract Id link to view your transaction in the source graph page. It will show the execution path of your contract. 
```